src/signatures.rs

```diff
@@ -1287,6 +1287,10 @@ signatures! {
     value = b"\x02\x00\x09\x00"
     value = b"\xD7\xCD\xC6\x9A"
 
+    format = EnhancedMetafile
```

Comment:
Values separated with spaces mean "or" and according to EMF spec, both signatures must match, which mean that you must use a comma between them. You can put both values on same line for better readability. In addition, there is an extra space between the value and offset and no space around the equal sign.

Reply:
Should also be sorted by format ident ascending within the same max signature size group (4-bytes here).